Sources/ForceSimulation/LinearCongruentialGenerator.swift

```diff
@@ -5,6 +5,7 @@ public protocol DeterministicRandomGenerator<Scalar> {
     associatedtype Scalar where Scalar: FloatingPoint & ExpressibleByFloatLiteral
     @inlinable static func next() -> Scalar
     @inlinable mutating func next() -> Scalar
+    @inlinable init()
```

Comment:
It's well after merge, but I wanted to ask - did you want to allow any mechanism in the protocol to provide a seed, rather than always starting in the same "position" on this noise space?

Wasn't 100% sure what you were after, but there's some interesting Xoroshiro256 implementations in swift, and I've fiddled with some nearby variants of what you're doing here (ported from C - http://github.com/heckj/Squirrel3/)

Reply:
Hi, I think visualization developers would like to have a stable-looking animation every time they create a view. Currently it's associated with type (`Float`/`Double`) and there's no extensibility about the randomness. 

A random protocol might be good for customization, but I don't want to:
- introduce a long type signatures, like `Simulation<Vector, Force, RandomGenerator>`
- or introduce erased types (which prevents specializing)
- or require the framework consumers to explicitly declare `extension SomeFloat: HasDeterministicGenerator { ... }`

If you have some suggestions I'll try to explore that. Thanks!

Reply:
Right - that makes perfect sense, thanks! No suggestions - what you've got is working great, I was noodling recently on more generalized usage of random with procedural generation efforts.